project.clj

```diff
@@ -14,5 +14,6 @@
                                    [lein-cljfmt "0.6.3"]
                                    [lein-cloverage "1.0.13"]
                                    [lein-kibit "0.1.6"]]
-                    :dependencies [[org.clojure/clojure "1.9.0"]]}}
+                    :dependencies [[org.clojure/clojure "1.9.0"]
+                                   [org.apache.logging.log4j/log4j-slf4j-impl "2.1"]]}}
```

Comment:
https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-slf4j-impl/2.1 has log4shell vulnerability. you might wanna force upgrade to https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-slf4j-impl/2.17.2.